server/secrets.py

```diff
@@ -1,8 +1,5 @@
 MBTA_V3_API_KEY = ''
-POSTGRES_ENABLED = True
-POSTGRES_DB = 'newtrains'
-POSTGRES_USER = 'ubuntu'  # Update to your Postgres user account if not using ubuntu
-POSTGRES_PASS = ''  # Add password if required; Uses account auth on linux
+LAST_SEEN_UPDATE = False
```

Comment:
Any reason not to make this true by default?

Reply:
Yes, actually—flask in debug mode spawns two processes for some reason, which would cause double/trampling updates. I'll note this in a comment